screens/Authentication/Login/index.tsx

```diff
@@ -14,10 +14,10 @@ GoogleSignin.configure({
   iosClientId: process.env.IOS_CLIENT_ID,
   offlineAccess: true,
 })
-type LoginProp = NativeStackNavigationProp<RootStackParamList, "Login">
+type SignProp = NativeStackNavigationProp<RootStackParamList, "Login">
 
```

Comment:
this should be as before `LoginProps`

usually the interface is the same as the screen/component name + Props